pkg/reconciler/trigger/trigger.go

```diff
@@ -136,16 +144,41 @@ func (r *Reconciler) reconcile(ctx context.Context, t *v1alpha1.Trigger) error {
 		// Everything is cleaned up by the garbage collector.
 		return nil
 	}
+	track := r.resourceTracker.TrackInNamespace(t)
+	dependencyAnnotation, ok := t.GetAnnotations()[v1alpha1.DependencyAnnotation]
+	if ok {
+		dependencyObjRef, err := getObjRefFromDependencyAnnotation(dependencyAnnotation)
+		if err != nil {
+			logging.FromContext(ctx).Error("Unable to unmarshal objectReference from dependency annotation of trigger:", zap.Error(err))
+			return err
```

Comment:
Do you mean remove this line "logging.FromContext(ctx).Error("Unable to unmarshal objectReference from dependency annotation of trigger:", zap.Error(err))"?

Reply:
Is it because func (r *Reconciler) Reconcile(ctx context.Context, key string) error{....}  method will  add logging with logging.FromContext(ctx).Error("Error reconciling Trigger", zap.Error(reconcileErr)) when func (r *Reconciler) reconcile(ctx context.Context, t *v1alpha1.Trigger) error{....} returns error? If we logging error in both methods, we will have duplicate logging? In this case, it looks like we also need to change the code when we reconcile broker and subscription in reconcile() method.